examples/README.md

```diff
@@ -2,46 +2,6 @@
 
 This examples are to help give a basic overview of what a repository using `rok8-scripts` might look like.
 
-## Jenkins-K8s-Plugin
+Click on any example to learn more about it.
```

Comment:
Should we put a listing here of the folders so they can click in both places?  That is probably a little annoying to do, but might make the UX a little better.

Reply:
The issue I find is that the links end up going stale as folders get added/renamed/deleted (as we see here). Maybe we can add EJ's TOC script from scriptlab at some point to manage this.

Reply:
Cool.  Seems good to me.